modules/pruning-images.adoc

```diff
@@ -66,8 +67,13 @@ status:
 <9> `startingDeadlineSeconds`: Start deadline for `CronJob`. This is an optional field.
 <10> `successfulJobsHistoryLimit`: The maximum number of successful jobs to retain. Must be `>= 1` to ensure metrics are reported. Defaults to `3` if not set.
 <11> `failedJobsHistoryLimit`: The maximum number of failed jobs to retain. Must be `>= 1` to ensure metrics are reported. Defaults to `3` if not set.
-<12> `observedGeneration`: The generation observed by the Operator.
-<13> `conditions`: The standard condition objects with the following types:
+<12> `logLevel`: Sets the level of log output for the pruner job. Defaults to  `Normal`.
+* `Normal`
+* `Debug`
+* `Trace`
```

Comment:
I think that a small description about each log level would be useful for the reader. If I were setting `Trace` or `TraceAll` log levels in my cluster I would like to know what the difference is between the two. In a critical situation this could help prevent multiple reproducers from being required in order to debug something.

Reply:
Agreed, and thank you. I checked the API and they aren't defined, so I'm not sure how to describe them at this point. I have a related issue/PR, so I'll merge this one as is an update the descriptions as soon as I get the information.